src/scancode/cli.py

```diff
@@ -216,6 +230,8 @@ def validate_formats(ctx, param, value):
               help=('Set <output_file> format <style> to one of the standard formats: %s '
                     'or the path to a custom template' % ' or '.join(formats)),
               callback=validate_formats)
+@click.option('--ignore', default='', multiple=True, show_default=True, metavar='<pattern>',
```

Comment:
A Nonetype object was throwing errors back when I added the ignore option to be parsed as a comma-separated list, I forgot to change it back in later commits. I'll change it in the next commit. :)